tests/system/parse.go

```diff
@@ -24,7 +24,7 @@ var (
 	// wait <duration> for session <session ID>
 	waitAction = regexp.MustCompile(`^wait\s+(.+)\s+(for|unless)\s+session\s+(.+)`)
 
-	// output <equals|equals_json|contains|regex> [file] *
+	// output <equals|equals_json|equals_contains|regex> [file] *
```

Comment:
Is this a mistake? I don't see `equals_contains` anywhere else

Reply:
oops